projects/display-project.js

```diff
@@ -1,31 +1,29 @@
 const queryParams = getURLParams();
-console.log("queryParams", queryParams);
-
-const isView = !queryParams.isView;
-console.log("isView", isView);
 
 let currentlyViewedProjectId = 0;
 
-if (isView) {
-  currentlyViewedProjectId = +queryParams.id;
+currentlyViewedProjectId = +queryParams.id;
+
+const viewedProject = getProjectById(currentlyViewedProjectId);
 
-  const viewedProject = getProjectById(currentlyViewedProjectId);
+const studentName = getStudentById(viewedProject.studentId);
 
-  const displayProjectContainer = document.getElementById(
-    "view-project-container"
-  );
+const studentDepartment = getDepartmentById(viewedProject.departmentId);
 
-  const displayTitleHolder = createElementWithAttributes("div", {
-    class: "project-display-title",
-  });
-  const displayContentHolder = createElementWithAttributes("div", {
-    class: "project-display-content",
-  });
+const studentSupervisor = getSupervisorById(viewedProject.superviorId);
 
-  displayTitleHolder.textContent = `Title - ${viewedProject.title}`;
-  displayContentHolder.textContent = viewedProject.content;
+const displayProjectContainer = document.getElementById("view-project-container");
 
-  displayProjectContainer.append(displayTitleHolder, displayContentHolder);
+const displayStudentHolder = document.getElementById("display-student-holder");
+const displayDepartmentHolder = document.getElementById("display-depatment-holder");
+const displaySupervisorHolder = document.getElementById("display-supervisor-holder");
+const displayTitleHolder = document.getElementById("display-title-holder");
+const displayContentHolder = document.getElementById("display-content-holder");
+const displayDateOfSubmissionHolder = document.getElementById("display-date-of-submission-holder");
 
-  console.log("displayProjectContainer", displayProjectContainer);
-}
+displayStudentHolder.innerHTML = `<b>Name:</b> ${studentName.name}`
```

Comment:
If you did one of the following above, then `studentName` will be a string, so you don't need to do `studentName.name` anymore.